src/Installers/Windows/AspNetCoreModule-Setup/ANCMVS.wixproj

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <OutputNamePrefix>aspnetcoremodulevs-</OutputNamePrefix>
+    <OutputNameSuffix>-win-$(Platform)</OutputNameSuffix>
+    <ProductNameFolder>Microsoft ASP.NET Core Module</ProductNameFolder>
+    <ProductNameShort>AspNetCoreModule.VS</ProductNameShort>
+    <OutputType>Package</OutputType>
+    <Cabinet>sfx_$(Platform).cab</Cabinet>
+    <ProjectGuid>C681D730-4505-42C6-9E6C-87F757C4FB32</ProjectGuid>
+    <HarvestDirectoryAutoGenerateGuids>true</HarvestDirectoryAutoGenerateGuids>
+    <HarvestDirectorySuppressSpecificWarnings>5150;5151</HarvestDirectorySuppressSpecificWarnings>
+    <HarvestDirectorySuppressRegistry>true</HarvestDirectorySuppressRegistry>
+    <HarvestSource>$(IntermediateOutputPath)fx\</HarvestSource>
+    <IsShipping>true</IsShipping>
+    <DefineConstants>$(DefineConstants);ProductNameFolder=$(ProductNameFolder)</DefineConstants>
+    <DefineConstants>$(DefineConstants);ProductNameShort=$(ProductNameShort)</DefineConstants>
+    <DefineConstants>$(DefineConstants);AspNetCoreSharedFrameworkSource=$(HarvestSource)</DefineConstants>
+    <NamespaceGuid>$(SharedFrameworkNamespaceGuid)</NamespaceGuid>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ToolsetInstallerNuspecFile>$(RepoRoot)\src\Installers\Windows\AspNetCoreModule-Setup\ANCMVS.nuspec</ToolsetInstallerNuspecFile>
+    <SkipCopyToArtifactsDirectory>true</SkipCopyToArtifactsDirectory>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <WixExtension Include="WixDependencyExtension">
```

Comment:
Is all of the wix stuff needed? I thought we just wanted to repackage the ANCM .msi's/assets that get created elsewhere